examples/pipelines/fl_cross_silo_literal/submit.py

```diff
@@ -282,10 +282,20 @@ def fl_cross_silo_internal_basic():
                 epochs=YAML_CONFIG.training_parameters.epochs,
                 # Dataloader batch size
                 batch_size=YAML_CONFIG.training_parameters.batch_size,
+                # Differential Privacy
+                dp=YAML_CONFIG.training_parameters.dp,
+                # DP target epsilon
+                dp_target_epsilon=YAML_CONFIG.training_parameters.dp_target_epsilon,
+                # DP target delta
+                dp_target_delta=YAML_CONFIG.training_parameters.dp_target_delta,
+                # DP max gradient norm
+                dp_max_grad_norm=YAML_CONFIG.training_parameters.dp_max_grad_norm,
                 # Silo name/identifier
                 metrics_prefix=silo_config.name,
                 # Iteration number
                 iteration_num=iteration,
+                # Total num of iterations
+                total_num_of_iterations=YAML_CONFIG.training_parameters.num_of_iterations,
```

Comment:
Nit; similarly, since "total_num_of_iterations" is only required for DP purpose, it might be slightly better to put this input arg together with those other DP-related parameters (line 285-292), and added brief docstring mention this arg is only required for DP.

Reply:
@garg-amit it seems you forgot to update this in submit.py in your last commit.